benchmarks/src/main/resources/k6-scripts/echo.proto

```
@@ -0,0 +1,46 @@
+syntax = "proto3";
+
+// Benchmark-lane copy of the in-repo gRPC echo contract, shipped to the k6 script directory so
+// the k6 gRPC client (grpc_unary.js) can load the descriptor and derive the real service/method
+// path. Source of truth: integration-tests/src/main/proto/echo.proto — the gateway relays this
+// service opaquely, so the two copies only need to agree on the package/service/method names and
+// the request/response shapes. Keep them in sync when the echo contract changes.
+package de.cuioss.sheriff.api.integration.grpc;
```

Buf (1.71.0)

[error] Files with package "de.cuioss.sheriff.api.integration.grpc" must be within a directory "de/cuioss/sheriff/api/integration/grpc" relative to root but were in directory "benchmarks/src/main/resources/k6-scripts".

(PACKAGE_DIRECTORY_MATCH)